src/polyfill.js

```diff
@@ -1,6 +1,7 @@
 import Promise from './index';
 import promiseFinally from './finally';
 
+var global;
```

Comment:
This shadows the global in Node JS and will cause it to not work correctly

Reply:
Suppressed the warning instead. Fixed :+1: